### ISSUE TITLE
GTK2 entries look like raised, clickable buttons

### ISSUE THREAD

**Issue by cassidyjames** [system76/pop-gtk-theme#5]
**Reporter info**

 ```
 * Distribution - Ubuntu GNOME 17.04
 * Gtk+ 3.0 version - 3.24
 * Desktop environment - GNOME Shell 
 ```

**Related Application and/or Shell theming (or name of widget)**
GTK2 text entries

**Actual issue**
Text entries look raised like clickable buttons

**Steps to reproduce (if you knew)**
Use a GTK2 app with text entries

**Expected behaviour**
Entries look like entries everywhere else, or at least not raised like click-able buttons

**Comment by cassidyjames** [comment]
![raised-entries](https://cloud.githubusercontent.com/assets/611168/25544619/7d4461f0-2c18-11e7-8995-2067128c5037.png)

**Comment by cassidyjames** [comment]
Confirmed that this is an issue on multiple GTK2 entries, not just that app. Here's a screenshot of GIMP
![gimp-raised-entries](https://cloud.githubusercontent.com/assets/611168/25544679/be2db7de-2c18-11e7-9e8f-abc8c2c24712.png)

**Comment by isantop** [comment]
Fixed in 4af6bbf
